ngraph_onnx/onnx_importer/backend.py

```diff
@@ -155,7 +156,11 @@ def __init__(self, ng_model_function, device='CPU'):  # type: (List[Function], s
         super(NgraphBackendRep, self).__init__()
         self.device = self._get_ngraph_device_name(device)
         self.ng_model_function = ng_model_function
-        self.runtime = ng.runtime(backend_name=self.device)
+        if device == 'IE:CPU':
```

Comment:
What about IE:GPU, IE:MIRIAD etc? we should search for IE string IMO.

Reply:
good point, done